datetime/src/commonMain/kotlin/com/revenuecat/purchases/kmp/datetime/CustomerInfo.kt

```diff
@@ -1,14 +1,14 @@
-package io.shortway.kobankat.datetime
+package com.revenuecat.purchases.kmp.datetime
```

Comment:
Any reason to have these in a separate module? We haven't been doing much modularization in `purchases-android` since we thought we didn't need it at least right now but would be good to know the reasoning.

Reply:
They add a dependency on `kotlinx-datetime` which I didn't want to impose on people if they don't use it already. Although it is probably the most popular datetime library, they might also be using [Klock](https://docs.korge.org/time/) or [Island Time](https://islandtime.io/), for instance.

Reply:
I do wonder if we want to be more opinionated with these, since it would make for a much easier structure... But I'm ok keeping it separate for now. At the end of the day, this is #customer-obsession

Reply:
Haha yea true. It's something I want to discuss with the team before the release!